### PR TITLE
Added a variable to the config file for Follow Us text and then added…

### LINKED ISSUE #48: Text in partial footer "Follow us" is hard coded
Should be a variable so template can be used in other languages

### PR DESCRIPTION
**Pull request by nathanbiller**
… this variable in place of the hardcoded english version.

Fixes #48

### CODE REVIEW

**Comment by nathanbiller** [comment]
The way I fixed it keeps the code cleanest long term, but anyone who updates their template without also adding in the variable to their config file will have a blank "Follow Us" section. I could have done it using an if statement so that if it wasn't specified in the config file it would default to the original English hardcoded version. I opted for the first because otherwise long term we'd just have bloated code to handle a safe fallback, but if there's a preference we could change this.

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
 {{- $quickLinks  := index $footer "quicklinks" }}
 {{- $socialMedia := index $footer "socialmedia" }}
 {{- $bulmaLogo   := index $footer "bulmalogo" }}
+{{- $followUs    := index $footer "socialmediatext" }}
```
Comment (StefMa) [inline layouts/partials/footer.html]: Please name the var `socialMediaTitle` and the "property" from the config file `socialmediatitle`

Comment (StefMa) [reply layouts/partials/footer.html]: Same for the config.yml of course 😉

**Comment by StefMa** [comment]
Yep. I totally agree with you here.
I don't want to have "fallback" versions.
Everything could be a breaking change...

In the longterm I also think about to versioning the theme and do "real releases"" instead of just letting users checkout the master branch and use it...
But this is not common in the `hugo` scene yet.

File: exampleSite/config.yaml [hunk 2]
```diff
@@ -113,6 +113,8 @@ params:
   footer:
     # Logo (from /images/logos/___)
     logo: fresh-white-alt.svg
+    # Social Media Title
+    socialmediatext: Follow Us
```
Comment (StefMa) [inline exampleSite/config.yaml]:
```suggestion
    socialmediatitlet: Follow Us
```

File: layouts/partials/footer.html (hunk 1 above)
Comment (StefMa) [inline layouts/partials/footer.html]:
```suggestion
{{- $socialMediaTitle    := index $footer "socialmediatitle" }}
```

File: layouts/partials/footer.html [hunk 3]
```diff
@@ -32,7 +33,7 @@ <h3>{{ .title }}</h3>
       <div class="column">
         <div class="footer-column">
           <div class="footer-header">
-            <h3>Follow Us</h3>
+            <h3>{{ $followUs }}</h3>
```
Comment (StefMa) [inline layouts/partials/footer.html]:
```suggestion
            <h3>{{ $socialMediaTitle }}</h3>
```

**Pull request merged**